### ISSUE TITLE
Service uses deprecated computed.readOnly for locales and defaultLocale

### ISSUE THREAD

**Issue by lupestro** [ember-intl/ember-intl#1584]
- [X] I am on the [latest](https://github.com/ember-intl/ember-intl/releases/latest) ember-intl version
- [X] I have searched the [issues](https://github.com/ember-intl/ember-intl/issues) of this repo and believe that this is not a duplicate

### Environment

- Ember Version: 3.27
- Ember CLI Version: 3.27
- Ember Intl Version: 5.7.0
- Browser(s): Chrome
- Node Version: 12

### Steps to Reproduce

Use ember-into in an application or add-on. inject the service in any module of the app.
Build and run with Ember 3.27
Note the deprecation messages in the console.

(I'm not sure what the alternatives to uses of computed.readOnly are or their implications, or I might have made this a PR rather than an issue. With a little guidance, it could still happen.)

**Comment by oliverlj** [comment]
I also have this issue.

here is the transpiled code when I run unit test:
```js
define("ember-intl/services/intl", ["exports", "@ember/application", "@ember/object", "@ember/object/evented", "@ember/debug", "@ember/array", "@ember/service", "@ember/runloop", "ember-intl/-private/formatters", "ember-intl/-private/utils/is-array-equal", "ember-intl/-private/utils/normalize-locale", "ember-intl/-private/utils/get-dom", "ember-intl/-private/utils/hydrate", "ember-intl/-private/store/container"], function(_exports, _application, _object, _evented, _debug, _array, _service, _runloop, _formatters, _isArrayEqual, _normalizeLocale, _getDom, _hydrate, _container) {
    "use strict";

    Object.defineProperty(_exports, "__esModule", {
        value: true
    });
    _exports.default = void 0;

    /**
   * Copyright 2015, Yahoo! Inc.
   * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
   */
    var _default = _service.default.extend(_evented.default, {
        /** @public **/
        formats: null,

        /**
     * Returns an array of registered locale names
     *
     * @property locales
     * @public
     */
        locales: _object.computed.readOnly('_translationContainer.locales'),

        /** @public **/
        locale: (0,
        _object.computed)('_locale', {
            set(_, localeName) {
                const proposed = (0,
                _array.makeArray)(localeName).map(_normalizeLocale.default);

                if (!(0,
                _isArrayEqual.default)(proposed, this._locale)) {
                    (0,
                    _object.set)(this, '_locale', proposed);
                    (0,
                    _runloop.cancel)(this._timer);
                    this._timer = (0,
                    _runloop.next)(()=>{
                        this.trigger('localeChanged');

                        this._updateDocumentLanguage(this._locale);
                    }
                    );
                }

                return this._locale;
            },

            get() {
                return (0,
                _object.get)(this, '_locale');
            }

        }),
...
```

locales is transpiled to computed.readonly

**Comment by longlho** [comment]
beta.2 has been published

**Comment by ijlee2** [comment]
I'll close this issue, as the deprecation found in [`5.7.0`](https://github.com/ember-intl/ember-intl/blob/v5.7.0/addon/services/intl.js#L30) had been fixed by using a getter in [`6.0.0-beta.2`](https://github.com/ember-intl/ember-intl/blob/v6.0.0-beta.2/addon/services/intl.js#L66-L68).
